src/main/java/io/fabric8/jenkins/openshiftsync/BuildDecisionHandler.java

```diff
@@ -103,12 +110,27 @@ public boolean shouldSchedule(Queue.Task p, List<Action> actions) {
           logger.info("Triggering Jenkins build on WorkflowJob " + wj.getFullName() + " due to running on kubernetes due to " + causeDescription(actions));
           return true;
         }
-        logger.info("Triggering OpenShift Build for WorkflowJob " + wj.getFullName() + " due to " + causeDescription(actions));
+
+        /* We are always finding the buildConfig by the the name of JenkinsJobName
```

Comment:
Remove this long comment and let's keep it into OpenshiftUtils.getBuildConfig() method if required.